Gemfile

```diff
@@ -28,7 +35,15 @@ gem 'jbuilder', '~> 2.5'
 # Use Redis adapter to run Action Cable in production
 # gem 'redis', '~> 3.0'
 # Use ActiveModel has_secure_password
-# gem 'bcrypt', '~> 3.1.7'
+gem 'bcrypt', '~> 3.1.11'
+
+gem 'bootstrap-sass', '~> 3.3.7'
+
+gem 'jquery-rails'
```

Comment:
Jquery was necessary for Bootstrap - I am not currently using it for anything else.

Reply:
I'm pretty sure Jonah used it for something.

Reply:
Yeah, I use JQuery in the flashcards and remote submissions now